apps/prairielearn/src/components/Navbar.html.tsx

```
@@ -9,6 +9,11 @@ import type { NavbarType, NavPage, NavSubPage } from './Navbar.types.js';
 import { ContextNavigation } from './NavbarContext.html.js';
 import { ProgressCircle } from './ProgressCircle.html.js';
 
+export function PreactNavbar(props: Parameters<typeof Navbar>[0]) {
+  // eslint-disable-next-line @eslint-react/dom/no-dangerously-set-innerhtml
+  return <div dangerouslySetInnerHTML={{ __html: Navbar(props).toString() }}></div>;
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)